xbmc/URL.cpp

```diff
@@ -799,6 +799,9 @@ std::string CURL::GetProtocolOption(const std::string &key) const
 
 void CURL::SetProtocolOption(const std::string &key, const std::string &value)
 {
+  // NFS doesn't support any protocol options
```

Comment:
Maybe move this to a new method `SupportsOptions`. It also needs to be called in SetProtocolOptions. Ideally it should be detected before calling the set but since it's spread everywhere I guess we have no alternative than returning early.

Reply:
named the method `AreProtocolOptionsProhibited`